cvat/settings/base.py

```diff
@@ -235,6 +235,7 @@ def generate_ssh_keys():
 
 # https://github.com/pennersr/django-allauth
 ACCOUNT_EMAIL_VERIFICATION = 'none'
+OLD_PASSWORD_FIELD_ENABLED = True
```

Comment:
Is this option used somehow?

Reply:
This option requires current user password for verification and used by django-rest-auth.